.github/workflows/backend-ci-cd.yml

```
@@ -0,0 +1,102 @@
+      - develop
+      - main
+      - develop-snapshot
+    paths:
+      - 'apps/backend/**'
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        service: [
+          'discovery-server','gateway', 'config-server', 'user-service', 'file-service', 'sync-service', 'common' ]
```

YAMLlint (1.35.1)

[error] too few spaces after comma

(commas)